test/e2e/specs/AutolendingPage.spec.js

```diff
@@ -88,4 +88,76 @@ describe('Autolending Page Spec', () => {
 			// assert toggle says on
 		});
 	});
+
+	describe('Autolend explanation text', () => {
+		it('Verify user balance below lendable amount', () => {
+			// Mock date and idle start time
+			cy.mock({
+				AutolendProfile: () => ({
+					cIdleStartTime: null,
```

Comment:
You should also mock out isEnabled and enableAfter in each of these tests, because otherwise they will be assigned random values which can change what text is displayed.
```
isEnabled: true,
enableAfter: 0,
```

Reply:
Done